src/main/java/fr/raluy/chocoratage/Main.java

```diff
@@ -2,30 +2,36 @@
 
 import fr.raluy.chocoratage.locking.SessionLocker;
 import org.jnativehook.GlobalScreen;
+import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import java.io.IOException;
 import java.net.URISyntaxException;
 import java.util.Optional;
 import java.util.logging.Level;
-import java.util.logging.Logger;
 
 public class Main {
-    private static final org.slf4j.Logger log = LoggerFactory.getLogger(Main.class);
+    private static final Logger log = LoggerFactory.getLogger(Main.class);
 
     public static void main(String... args) throws IOException, URISyntaxException {
+        configureJNativeHookLogger();
+
         Config.parse(args);
 
-        displayOsInfo();
-        configureJNativeHookLogger();
+        if (Config.isDebugMode()) {
+            displayOsInfo();
+        }
 
         KeyLogger keyLogger = new KeyLogger();
         Os os = Optional.ofNullable(Config.getForcedOs()).orElseGet(() -> Os.guess());
+        SessionLocker sessionLocker = new SessionLocker(os);
+        keyLogger.addListener(sessionLocker);
+
         if (Config.isDebugMode()) {
             log.info("OS set to {}", os);
+            log.info("Forbidden phrases:", os);
```

Comment:
os gets logged twice?

Reply:
It's a bad case of copy paste, there's not even a `{}`. Should be `log.info("Forbidden phrases:");`